doc/source/ray-more-libs/ray-collective.rst

```diff
@@ -351,7 +351,7 @@ The following links provide helpful resources on how to efficiently leverage the
 
 
 * `More running examples <https://github.com/ray-project/ray/tree/master/python/ray/util/collective/examples>`_ under ``ray.util.collective.examples``.
-* `Scaling up the Spacy Name Entity Recognition (NER) pipeline <https://github.com/explosion/spacy-ray>`_ using Ray collective library.
+* `Scaling up the spaCy Named Entity Recognition (NER) pipeline <https://github.com/explosion/spacy-ray>`_ using Ray collective library.
```

Comment:
Unfamiliar with casing for this library.

Reply:
<img width="966" height="397" alt="image" src="https://github.com/user-attachments/assets/823d832d-910a-4151-9cd5-569a5418d6ca" />
you could see their repo(https://github.com/explosion/spacy-ray), I'm confused too...